### ISSUE TITLE
SafeAreaView not working

### ISSUE THREAD

**Issue by Dr1992** [octopitus/rn-sliding-up-panel#133]
### Issue Description

I am trying to put SlidingUpPanel inside SafeAreaView but is not working.
SlidingUpPanel looks like without SafeAreaView in bootom screen and it is not cool on iPhone X

### Steps to Reproduce / Code Snippets / Screenshots

![image](https://user-images.githubusercontent.com/17261405/59968881-0492e400-9518-11e9-9836-73cf276d1748.png)
![image](https://user-images.githubusercontent.com/17261405/59968886-12e10000-9518-11e9-9c1e-90a91d25f002.png)
![image](https://user-images.githubusercontent.com/17261405/59968893-23917600-9518-11e9-9a8d-cee7b1b22511.png)

---
### Environment
* Version: FILL THIS OUT
* React Native version:0.59.5
* React Native CLI: 2.0.1
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator iPhone X

**Comment by octopitus** [comment]
Are you using the [built-in SafeAreaView](https://facebook.github.io/react-native/docs/0.59/safeareaview)?

**Comment by octopitus** [comment]
If you are using the built-in one, consider switching to [community's SafeAreaView](https://github.com/react-native-community/react-native-safe-area-view). Works as expected.

**Comment by Dr1992** [comment]
Yeah. It worked fine but anyway it should be able to work with import { SafeAreaView } from 'react-navigation'

**Comment by jamesholcomb** [comment]
Did you try setting the backdropStyle.top to a negative offset to account for your header and statusbar?

**Comment by joelrorseth** [comment]
I just found your issue while trying to debug my own problems with SafeAreaView. I couldn't seem to calculate the correct draggableRange, to allow my header to stay within the top and bottom safe areas. After a lot of trial and error, I found that the following worked. Hopefully this helps somebody else:

```
import { getInset } from 'react-native-safe-area-view';
const { width, height } = Dimensions.get('window');
const landScape = width > height;
const topPadding = getInset('top', landScape);
const bottomPadding = getInset('bottom', landScape);
```

```
<SlidingUpPanel
    draggableRange={{
        top: height - topPadding - bottomPadding,
        bottom: PANEL_HEADER_HEIGHT,
    }}
    height={height - topPadding}
/>
```
This manages to keep my header (in purple) within the safe area. Seems to work with the community and built in versions of SafeAreaView. The height is set so that the fully dragged up view aligns its bottom with the bottom of screen.

![Simulator Screen Shot - iPhone X - 2019-08-14 at 18 58 08](https://user-images.githubusercontent.com/10015052/63062486-26c93280-bec7-11e9-8aeb-c8bebb1cb608.png)
![Simulator Screen Shot - iPhone X - 2019-08-14 at 18 58 05](https://user-images.githubusercontent.com/10015052/63062489-2c267d00-bec7-11e9-86e3-1d9a0393e320.png)
